docs/docs/recipes/integrate-logto/traditional.mdx

```diff
@@ -200,6 +203,16 @@ exports.refreshTokens = async (refreshToken) => {
 
 ## Sign in
 
+<SignInFlowSummary />
```

Comment:
should we also need to redirect to Logto during sign-out?

Reply:
Yes, this is in the TODO, after LOG-2651 is completed, I'll add a link to that part.